File: src/MasterDetail.Web/app/js/controllers/statisticsController.js
define(["app"], function (app) {
    app.register.controller("StatisticsController",
        ["$scope", function ($scope) {
            ttTools.logger.info("Freakin' cool stats!");

            // NOTE: Hard-coded sample values - go and fetch via Web API/localStorage
            $scope.pieSeries = [{
                type: "pie",
                startAngle: 150,
                data: [{
                    category: "Asia",
                    value: 53.8,
                    color: "#9de219"
                }, {
                    category: "Europe",
                    value: 16.1,
                    color: "#90cc38"
                }, {
                    category: "Latin America",
                    value: 11.3,
                    color: "#068c35"
                }, {
                    category: "Africa",
                    value: 9.6,
                    color: "#006634"
                }, {
                    category: "Middle East",
                    value: 5.2,
                    color: "#004d38"
                }, {
                    category: "North America",
                    value: 3.6,
                    color: "#033939"
                }]
            }];

            $scope.columnSeries = [
            {
                name: "Total Sales",
                data: [56000, 63000, 74000]
            },
            {
                name: "Discounted Sales",
                data: [52000, 34000, 23000]
            }];
        }]);
});

File: src/myProducts.Web/Api/Controllers/StatisticsController.cs
using System;
using System.Collections.Generic;
using System.Web.Http;

namespace MyProducts.Web.Api.Controllers
{
    [RoutePrefix("api/statistics")]
    public class StatisticsController : ApiController
    {
        [Route("distribution")]
        public dynamic GetDistribution()
        {
            var data = new List<dynamic>
                {
                    new
                        {
                            category = "Asia",
                            value = 33.8
                        },
                    new
                        {
                            category = "Europe",
                            value = 36.1
                        },
                    new
                        {
                            category = "Latin America",
                            value = 11.3
                        },
                    new
                        {
                            category = "Africa",
                            value = 9.6
                        },
                    new
                        {
                            category = "Middle East",
                            value = 5.2
                        },
                    new
                        {
                            category = "North America",
                            value = 3.6
                        }
                };

            return data;
        }


        [Route("sales")]
        public dynamic GetSales()
        {
            var data = new List<dynamic>
                {
                    new
                        {
                            key = "Total Sales",
                            values = new[]{new dynamic[]{"A", 56000}, new dynamic[]{"B", 63000}, new dynamic[]{"C", 74000}}
                        },
                    new
                        {
                            key = "Discounted Sales",
                            values = new[]{new dynamic[]{"D", 52000}, new dynamic[]{"E", 34000}, new dynamic[]{"F", 23000}}
                        }
                };

            return data;
        }
    }
}